src/components/Sum.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a = 0, b = 0 }) => (
```

Comment:
```suggestion
const Sum = ({ a, b }) => (
```

Reply:
why you think so? 

what is the difference,